packages/webpack-config-terra/src/aggregate-themes/README.md

```diff
@@ -0,0 +1,127 @@
+# Theme Aggregation
```

Comment:
It might be worth throwing some disclaimers onto this doc page and redirects to the updated theme strategy incase people end up here without enough context to know this approach has been deprecated

Reply:
should I just delete the readme?

Reply:
I will, people shouldn't be using this directly.